scripts/deploy_bug.py

```diff
@@ -66,57 +72,63 @@ def main(argv):
 
     # Convert to int and drop any that don't
     tags = [int(tag) for tag in tags if tag.isdigit()]
+    return tags
```

Comment:
I figured I'd extract this into a separate function to make `main` more readable.